javascript/node/selenium-webdriver/chrome.js

```diff
@@ -727,6 +737,37 @@ class Driver extends webdriver.WebDriver {
         new command.Command(Command.LAUNCH_APP).setParameter('id', id),
         'Driver.launchApp()');
   }
+  
+  /**
+   * Schedules a command to get Chrome network emulation settings.
+   * @return {!promise.Thenable<T>} A promise that will be resolved
+   *     when network emulation settings are retrievied.
+   */
+  getNetworkConditions() {
+    return this.schedule(
+        new command.Command(Command.GET_NETWORK_CONDITIONS),
+        'Driver.getNetworkConditions()');
+  }
+
+  /**
+   * Schedules a command to set Chrome network emulation settings.
+   * @param {number} latency Additional latency (ms).
+   * @param {number} download_throughput Maximal aggregated download throughput. 
+   * @param {number} upload_throughput Maximal aggregated upload throughput.
+   * @return {!promise.Thenable<void>} A promise that will be resolved
+   *     when network emulation settings are set.
+   */
+  setNetworkConditions(latency, download_throughput, upload_throughput) {
```

Comment:
After some further testing, it looks like all of the fields are required.  If you remove a field (latency for example), the chromedriver throws an error (see below).  Do you think I should still switch to a config object?

Thanks,
Dave

WebDriverError: unknown error: invalid 'network_conditions' is missing 'latency'
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 10.0.14393 x86_64)
    at Object.checkLegacyResponse (C:\Users\t-dan\Desktop\temp\node_modules\selenium-webdriver\lib\error.js:517:15)
    at parseHttpResponse (C:\Users\t-dan\Desktop\temp\node_modules\selenium-webdriver\lib\http.js:516:11)
    at doSend.then.response (C:\Users\t-dan\Desktop\temp\node_modules\selenium-webdriver\lib\http.js:432:13)
    at process._tickCallback (internal/process/next_tick.js:109:7)
From: Task: Driver.setNetworkConditions(5, 512000, 512000)
    at thenableWebDriverProxy.schedule (C:\Users\t-dan\Desktop\temp\node_modules\selenium-webdriver\lib\webdriver.js:816:17)
    at thenableWebDriverProxy.setNetworkConditions (C:\Users\t-dan\Desktop\temp\node_modules\selenium-webdriver\chrome.js:767:17)
    at Object.<anonymous> (C:\Users\t-dan\Desktop\temp\app.js:7:8)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:425:7)